res/control-roles.py

```
@@ -0,0 +1,292 @@
+    params = {"pageSize": page_size}
+    if name is not None:
+        params["name"] = name
+    if status is not None:
+        params["status"] = STATUSES[status]
+
+    roles = []
+    current = 0
+    while True:
+        current += 1
+        params["current"] = current
+        response = requests.get(
```

Ruff (0.16.3)

[error] Probable use of `requests` call without timeout

(S113)